projects/aca-content/src/lib/components/favorite-libraries/favorite-libraries.component.ts

```diff
@@ -64,6 +70,15 @@ export class FavoriteLibrariesComponent extends PageComponent implements OnInit
   ngOnInit() {
     super.ngOnInit();
 
+    this.extensions
```

Comment:
can be moved to base class, also, why removing certain actions in code? should be removed from the extensions json file?